packs/october.js

```diff
@@ -0,0 +1,39 @@
+  'unused-javascript': "Consider reviewing the [plugins](https://octobercms.com/plugins?search=javascript) that load unused JavaScript in the web page. To identify plugins that add unnecessary JavaScript, run [code coverage](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) in Chrome DevTools. Identify the theme/plugin responsible from the URL of the script. Look for plugins with many scripts with lots of red in code coverage. A plugin should only add a script if it is actually used on the web page.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve their site by enabling long caching in the context of the October CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-long-cache-ttl': "Read about [preventing unnecessary network requests with the HTTP Cache](https://web.dev/http-cache/#caching-checklist). There are many [plugins](https://octobercms.com/plugins?search=Caching) that can be used to speed up the caching.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve site performance by optimizing images, in the context of the October CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-optimized-images': "Consider using an [image optimization plugin](https://octobercms.com/plugins?search=image) that compresses the images while retaining the quality.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance via enabling text compression in the context of the October CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-text-compression': "Enable text compression in the web server configuration.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by using responsive images in the context of the October CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-responsive-images': "Upload images directly through the [media manager](https://octobercms.com/docs/cms/mediamanager) to ensure that the required image sizes are available and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). Avoid using `Full Size` images unless the dimensions are adequate for their usage. [Web Fundamentals Images](https://developers.google.com/web/fundamentals/design-and-ux/responsive/images).",
```

Comment:
```suggestion
  'uses-responsive-images': "Upload images directly with (----or in?---) the [media manager](https://octobercms.com/docs/cms/mediamanager) to ensure that the required image sizes are available and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). Avoid using `Full Size` images unless the dimensions are adequate for their usage. [Web Fundamentals Images](https://developers.google.com/web/fundamentals/design-and-ux/responsive/images).",
```

(this sentence is long and difficult to follow/parse. can you break it up?)